scalalib/overrides-2.11/scala/Array.scala

```diff
@@ -0,0 +1,531 @@
+/*                     __                                               *\
```

Comment:
What has been overriden in this file? Is it possible to make it work without it? We should avoid overrides as much as possible.